docgenhtml/plugins/org.polarsys.capella.docgen/src/org/polarsys/capella/docgen/util/CapellaComponentServices.java

```diff
@@ -150,6 +165,80 @@ public static String componentExchangeToTableLine(Component currentComponent, Co
 
 		return buffer.toString();
 	}
+	
+	public static String physicalLinkToTableLine(PhysicalLink physicalLink, String projectName, String outputFolder) {
+		StringBuilder buffer = new StringBuilder();
+		buffer.append(CapellaServices.TR_OPEN);
+		buffer.append("<td id=\"" + CapellaServices.getAnchorId(physicalLink) + "\">");
+		buffer.append(CapellaServices.getImageLinkFromElement(physicalLink, projectName, outputFolder));
+		buffer.append(" ");
+		buffer.append(CapellaServices.getHyperlinkFromElement(physicalLink));
+		buffer.append(CapellaServices.TD_CLOSE);
+
+		List<AbstractPhysicalLinkEnd> linkEnds = physicalLink.getLinkEnds();
+		buffer.append(CapellaServices.TD_OPEN);
+		if (!linkEnds.isEmpty()) {
+			buffer.append(CapellaServices.UL_OPEN);
+			for (AbstractPhysicalLinkEnd end : linkEnds) {
+				EObject container = end.eContainer();
+				buffer.append(CapellaServices.LI_OPEN);
+				buffer.append(CapellaServices.getImageLinkFromElement(container, projectName, outputFolder));
+				buffer.append(" ");
+				buffer.append(CapellaServices.getHyperlinkFromElement(container));
+				buffer.append(CapellaServices.LI_CLOSE);
+			}
+			buffer.append(CapellaServices.UL_CLOSE);
+		}
+		buffer.append(CapellaServices.TD_CLOSE);
```

Comment:
code duplicated 4 times 
should be in a separate method

Reply:
This particular comment was dealt with
All other comments are about gen code, which is automatically generated when building after the fcore was updated